### ISSUE TITLE
Unable to find installed modules

### ISSUE THREAD

**Issue by diviyank** [elarivie/linter-mypy#16]
Hi, thanks for the great package. 
I seem to have an unexpected behaviour : linter-mypy does not seem to find any installed package, even through I gave the right executable (referring to a conda environment having all the installed packages).

Do you have any ideas about where I could have messed up?

Thanks a lot,
Diviyan

**Comment by elarivie** [comment]
@Diviyan-Kalainathan 
Hello, thanks for your feedback

I am not familiar with [Conda](https://en.wikipedia.org/wiki/Conda_(package_manager)) environment... could you provide more details, what error exactly do you have?
Which package exactly are not found?

Do you obtain valid output when launching the following two commands in a terminal (note use the path to python that you provided in the linter-mypy settings):
```
python -m mypy --version
python -m mypy --help
```
I guess that the python executable that you provide in the settings is the one managed by Conda? Did you try to provide the full path to it?

Did you try using the setting "Mypy Path"?

If you manually launch in a terminal a Mypy analyse of your code do you obtain the same error?

How do you launch atom? does your environment variable PATH modified by Conda is available to it?

Could you provide some kind of script which install & setup a Conda environment and reproduce the issue that you have.

**Note:** Even though it is not the same environment manager, your question really looks like #6.

**Comment by diviyank** [comment]
Thanks for the answer, the issue is concerning all non-default packages installed in the environment : 

![screenshot-mypy](https://user-images.githubusercontent.com/20682356/37512758-524aedaa-2903-11e8-896e-0d383bf8b264.png)

Here's the output of the terminal commands, nothing seems to be wrong: 
```
$ /home/diviyan/miniconda2/envs/py35/bin/python -m mypy --version
mypy 0.570
$ /home/diviyan/miniconda2/envs/py35/bin/python -m mypy --help
usage: mypy [-h] [-v] [-V] [--python-version x.y] [--platform PLATFORM] [-2]
            [--ignore-missing-imports]
            [--follow 
            ...
```
- I did try both paths: the path with '~' didn't find mypy so I'm using the full path right now (this related to #6 but I had it fixed already)
- I just tried changing the mypy path in the setting, nothing changes. 

I noticed that the issue doesn't comes from linter-mypy but from mypy itself! Running mypy in a terminal gives me the same errors. I'm going to investigate in that direction. 

Thanks a lot for your feedback !
